test/CommonAdapterTestTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cache-storage-adapter-test for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cache-storage-adapter-test/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cache-storage-adapter-test/blob/master/LICENSE.md New BSD License
+ */
+
+namespace LaminasTestTest\Cache\Storage\Adapter;
+
+use Laminas\Cache\Storage\Adapter\AdapterOptions;
+use LaminasTest\Cache\Storage\Adapter\CommonAdapterTest;
+
+/**
+ * @group      Laminas_Cache
+ */
+final class CommonAdapterTestTest extends CommonAdapterTest
+{
+    public function setUp()
```

Comment:
```suggestion
    public function setUp(): void
```

This will add forward compatibility to phpunit 9.5

Reply:
Can't do this, the CI runs on 5.6 too

Reply:
So drop it? Dropping PHP version in minors is okay in laminas packages. Done that in laminas-cache 2.11.x aswell.

Reply:
But we can do that later on aswell.